Vagrantfile

```diff
@@ -22,6 +22,7 @@ MEMORY         = settings['memory']
 STORAGECTL     = settings['vagrant_storagectl']
 ETH            = settings['eth']
 DOCKER         = settings['docker']
+VERBOSE        = settings['verbose']
```

Comment:
You need to update `vagrant_variables.yml.sample` and `vagrant_variables.yml.atomic` to reflect this config option